website/project/metadata/utils.py

```diff
@@ -21,8 +21,8 @@ def serialize_draft_registration(draft, auth=None):
         'registration_schema': serialize_meta_schema(draft.registration_schema),
         'initiated': str(draft.datetime_initiated),
         'updated': str(draft.datetime_updated),
-        'config': draft.config or {},
-        'flags': draft.flags,
+        'config': {}, # or draft.config or {},
```

Comment:
@barbour-em as a note, this formatting is a flake8 exception. Two spaces should proceed all inline comments.